auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -9,19 +9,11 @@
           <div class="hero-banner__cta-btns mb-2">
             <!-- Authenticated -->
             <div v-if="userProfile" class="cta-btns-authenticated">
-              <name-request-button :isWide="true" />
-              <v-btn large dark color="bcgovblue" class="cta-btn-auth font-weight-bold"
+              <v-btn large color="#003366" class="cta-btn-auth font-weight-bold white--text mr-4"
                      @click="goToManageBusinesses()">
-                Incorporate a Named Benefit Company
-              </v-btn>
-              <v-btn large dark color="bcgovblue" class="cta-btn-auth font-weight-bold"
-                     @click="goToManageBusinesses(true)">
-                Incorporate a Numbered Benefit Company
-              </v-btn>
-              <v-btn large color="bcgovgold" class="cta-btn-auth font-weight-bold"
-                     @click="goToManageBusinesses()">
-                Manage an Existing Business
+                Manage my Business
               </v-btn>
+              <name-request-button :isWide="true" />
```

Comment:
Use PascalCase (so that you can F12 the name to go to its declaration).

Reply:
In theory, you don't need to declare this as a binding, since the value is static (delete `:`).

Reply:
Also in the other files (at least for the components you're touching).

Reply:
I tried moving ":", but it wouldn't work.

Reply:
Try `isWide=true`.

Reply:
`isWide=true` works for `name-request-button` component. For `learn-more-btn` component, if we just use `isWide=true` and `:redirect="learnMoreUrl", then it will work. But it looks inconsistent.

![image](https://user-images.githubusercontent.com/51458187/184704203-b5edc803-0fb7-476f-98c3-d0aac281d832.png)

Reply:
I think `redirect-url` needs the binding (:) since the value is a variable.

Reply:
Sure. I'll use `isWide=true` and `:redirect="learnMoreUrl"`. Thanks.